### DISCUSSION TITLE
installing wsl is causing my system to reboot in loop with blue screen of death

### DISCUSSION THREAD

**Discussion by Adilkhan04** [community/community#52357] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Show & Tell

### Body

For something in my education I have to install wsl in order to use linux on my windows system. I tried to install it in PowerShell using following command:

wsl --install

So far so good, no error message and powershell tells me to reboot the sytem.

But when I reboot the system I get a blue screen of death and the pc keeps rebooting into the same bsod until I can get back into the sytem by restoring from a restore point. However after that wsl isn't installed.

The blue screen of death code is SECURE_PCI_CONFIG_SPACE_ACCESS saying AMDRyzenMasterDriver.sys failed.

i again have to restore my system to a previous point to recover from this error but I'm unable to install WSL no matter what I try when it reboots, it goes into BSOD loop everytime 

 I have a HP OMEN Laptop 15-en0xxx with AMD Ryzen 7 4800H with Nvidea Graphics with Windows 11 Home Insider 22H2.

my end goal is to install docker.

here is my minidump file - https://drive.google.com/file/d/1MPbcbKuEz28Y_4SA0G7kNffI05JtiNSq/view?usp=sharing

**Comment by SMelidoni** [comment, marked as answer]
Hi @adilkhan69 👋🏽

It seems that the issue you're facing is related to the AMDRyzenMasterDriver.sys, which is a driver associated with the AMD Ryzen Master Utility. This utility is known to sometimes cause conflicts with WSL and Virtualisation on Windows systems.

To resolve this issue, follow these steps:

1. Uninstall the AMD Ryzen Master Utility from your system. You can do this through "Apps & Features" or "Add or Remove Programs" in the Control Panel.
2. Reboot your system to ensure the driver is no longer loaded.
3. Install WSL again using the `wsl --install` command in PowerShell.
4. Reboot your system as prompted by the installation process.
5. If WSL is successfully installed and your system is stable, you can proceed to install Docker.

Note: If you still require the AMD Ryzen Master Utility, you may try installing the latest version of the software after you have successfully installed WSL and Docker. However, be aware that it might still cause conflicts with WSL, and you may need to uninstall it again if problems persist.

If the issue continues, consider updating your BIOS and system drivers to their latest versions. You can find the appropriate drivers and BIOS updates on the HP Support website.

I hope this helps!

**Reply by Adilkhan04** [reply]
Thanks you, this solved my issue now 
I was trying to figure it out myself for a very long time 
Thanks again !

**Reply by SMelidoni** [reply]
Hi @adilkhan69, 

That's great to hear! Glad it solved your problem 🎉

Please could you mark my answer as the accepted solution so that others can easily find the solution to this problem? Thanks!
